cypress/integration/ui/searchbox.spec.js

```diff
@@ -4,13 +4,16 @@ describe('Search box', () => {
   });
 
   it("is present", () => {
-    cy.menu("Configuration");
-    cy.get('div[class=panel-heading]').first().click();
+    cy.menu("Configuration", "Providers");
+    // Unsure what the point of the below was... but doesn't exist in any of the
+    // "Configuration -> *" menus, so causes the spec to fail if it exists
+    //
+    // cy.get('div[class=panel-heading]').first().click();
```

Comment:
😕

Reply:
panel-heading is a listnav, I think the point was to activate the first accordion

Reply:
Okay, I will try looking into this more then.  Was definitely not obvious that this is what was being done.

Reply:
So....

This took me a bit to figure out a passable solution (for now).

What was done previously when going to "Configurations" (by itself) was take you to a "foreman" page, where the the `div[class=panel-heading]` does exist, but this page doesn't seem to exist from the menu in `jansa++`.

I knew that `Compute -> Infrastructure -> Virtual Machines` did have the `listnav` in it, so I decided that was a good of place as any to start.

However, in addition, it seems like the `menu` command with this new menu doesn't play nicely when you got to a menu item that is part of the existing menu you are part of, due to the fact the menu is already pre-expanded in that case, so the page navigation gets all borked as a result.  This is a problem in this case, since the default landing page after logging in seems to be `Compute -> Infrastructure -> Providers` (or something), which then has the `Infrastructure` menu already open and re-clicking it as part of the command actually minimizes the menu instead of expanding it.

As a work around (for now), I just went to `Overview -> Dashboard` first, then to the `Compute -> Infrastructure -> Virtual Machines` default the menu to "unexpanded" for this spec.  Kinda sucks since it is an extra page load, but seems to work and at least allow the test to pass.

Reply:
Ok, sounds like we need to fix the menu helper then :).

But yeah, the test should really have been written like...

```
  cy.menu("Configuration", "Providers").accordion("Providers").search_box();
```

(and since then https://github.com/ManageIQ/manageiq-ui-classic/pull/6782 happened)

Reply:
Anyway, if you just use Configuration Providers and not ensure the right listnav, this test can be just 2 lines and no bugs :)